thewalrus/quantum/gaussian_checks.py

```diff
@@ -157,3 +157,34 @@ def fidelity(mu1, cov1, mu2, cov2, hbar=2, rtol=1e-05, atol=1e-08):
     # as opposed to 0.25 in Brask. This is because this function returns the square
     # of their fidelities.
     return f
+
+
+def is_symplectic(S, rtol=1e-05, atol=1e-08):
+    """Checks if the matrix is symplectic
+    Args:
+        S (array): a matrix
+        rtol (float): the relative tolerance parameter used in `np.allclose`
+        atol (float): the absolute tolerance parameter used in `np.allclose`
+    Returns:
+        (bool): whether the given matrix is symplectic
```

Comment:
```suggestion
        atol (float): the absolute tolerance parameter used in `np.allclose`

    Returns:
        (bool): whether the given matrix is symplectic
```

Reply:
Unlike in markdown, in ReST, double backticks are required to represent 'code text'